library/groovy

```diff
@@ -4,72 +4,72 @@ GitRepo: https://github.com/groovy/docker-groovy.git
 
 # Groovy 3
 
-Tags: 3.0.10-jdk8, 3.0-jdk8, jdk8
+Tags: 3.0.10-jdk8, 3.0-jdk8, jdk8, 3.0.10-jdk8-jammy, 3.0-jdk8-jammy, jdk8-jammy
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: 89fdd8ea47368a20609a57e90bb8b9567e9b1e07
+GitCommit: 4d9411a045265bb6c8af2a64876419301a9c5ee4
 Directory: jdk8
 
-Tags: 3.0.10-jdk11, 3.0-jdk11, jdk11
+Tags: 3.0.10-jdk11, 3.0-jdk11, jdk11, 3.0.10-jdk11-jammy, 3.0-jdk11-jammy, jdk11-jammy
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 89fdd8ea47368a20609a57e90bb8b9567e9b1e07
+GitCommit: 4d9411a045265bb6c8af2a64876419301a9c5ee4
 Directory: jdk11
 
-Tags: 3.0.10-jdk11-alpine, 3.0-jdk11-alpine, jdk11-alpine
+Tags: 3.0.10-jdk11-alpine, 3.0-jdk11-alpine, jdk11-alpine, 3.0.10-jdk11-alpine-jammy, 3.0-jdk11-alpine-jammy, jdk11-alpine-jammy
```

Comment:
`alpine-jammy` :question: :eyes: 

Looks like some bizzare tags got added.

Reply:
I must have really messed up my string replacements. Thank you. I'll get this cleaned up.